airbyte-integrations/connectors/source-gas-prices/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
+    "todo-field-name": "todo-abnormal-value"
+  }
```

Comment:
if you have incremental sync you need correct this file to test some abnormal states